tools/proofers/report_bad_page.php

```diff
@@ -50,9 +50,15 @@
     echo "<input type='hidden' name='submitted' value='true'>\n";
 
     echo "<p><b>" . _("Reason") . ":</b> ";
-    echo "<select name='reason'>";
+    echo "<select name='reason' required>";
     foreach($PAGE_BADNESS_REASONS as $i => $reason)
-        echo "<option value='$i'>$reason</option>";
+    {
+        if ($i == 0) {
+            echo "<option value=''></option>";
```

Comment:
Can we do one-better and make this option disabled so they can't select it? I forget how browsers treat fields like that.

Reply:
I can remove the empty option and default to the first reason (Bad Image)
![image](https://user-images.githubusercontent.com/1526781/85814207-0b57c680-b72b-11ea-8341-e081e428958d.png)
I can disable the selection for this option too as you suggest, but it is selected by default so it doesn't prevent submitting. Let me know what you think.

Reply:
I definitely woiuldn't set one of the options as a default. We've seen what happens when we do that, and it's why we don't want a default value set. The user should be forced to choose one of the options. Can the Submit button be disabled unless an option is chosen?

Testing in the sandbox, showing the error message, and not allowing the report to be submitted unless a value is chosen seems to work fine.

Reply:
I can add that logic, but that would involve adding JavaScript and I think it achieves the same thing we have here with the required attribute. Let me know though if you want me to pursue that option.

Reply:
This is actually working great as-is. Thank you for fixing it!